source/extensions/common/wasm/wasm.h

```diff
@@ -245,6 +245,12 @@ struct AsyncClientHandler : public Http::AsyncClient::Callbacks {
   Http::AsyncClient::Request* request;
 };
 
+template <typename T> struct Global {
```

Comment:
Another naive question
Where can I find a inherited Global<T> ?

Reply:
in wavm/wavm.cc.  That is the private VM specific implementation of the Global<T> class.

Reply:
Thanks! I didn't find it because github doesn't render large diff.